quickwit-doc-mapping/src/default_doc_mapper/default_mapper.rs

```diff
@@ -253,13 +253,27 @@ impl DocMapper for DefaultDocMapper {
         Ok(document)
     }
 
-    fn query(&self, _request: &SearchRequest) -> Result<Box<dyn Query>, QueryParserError> {
-        todo!()
+    fn query(&self, request: &SearchRequest) -> Result<Box<dyn Query>, QueryParserError> {
+        //TODO: This is just a placeholder implementation
+        // allowing us to test few things up front.
+        let schema = self.schema();
+        let default_fields = vec![schema.get_field("body").unwrap()];
```

Comment:
you can use the default_fields from the doc mapper directly! it's a hash map with keys as field name and values as field.

Reply:
(I think we should ditch it though to be coherent with the previous discussions... Only manipulating "field names" and no Field outside of the scale of a spit has the merit of allowing schema change. It does not matter too too much however.

Reply:
yes you're right, let's keep the field names only, so a Vec<String> is good enough here.

Reply:
Thanks @fmassot  :+1: this has the merit of making my test fully independent in the meantime.